o-fish-ios/Views/ReportFlow/ReportNavigationRootView.swift

```diff
@@ -95,6 +98,9 @@ struct ReportNavigationRootView: View {
     private func saveAlertClicked() {
         print("Saving report in Realm")
         report.save()
+        DispatchQueue.main.asyncAfter(deadline: .now() + 0.1) {
```

Comment:
timer looks a bit hacky

Reply:
It's hack, because swiftUI doesn't have possibility show alert one after another.